Task3,4,5

Khol2721 opened the pull request:
homework 3, 4, 5

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

blnvfeip reviewed (commented):
проект не собирается

Khol2721 committed:
Delete app directory

Khol2721 committed:
Delete gradle/wrapper directory

Khol2721 committed:
Delete .gitignore

Khol2721 committed:
Delete README.md

Khol2721 committed:
Delete build.gradle

Khol2721 committed:
Delete gradle.properties

Khol2721 committed:
Delete gradlew

Khol2721 committed:
Delete gradlew.bat

Khol2721 committed:
Delete settings.gradle

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Add files via upload

Khol2721 committed:
Fix

Khol2721 committed:
Delete Рабочий стол - Ярлык.lnk

Khol2721 committed:
fix

Khol2721 committed:
Merge branch 'task3,4,5' of https://github.com/Khol2721/fefu-android-base-repo into task3,4,5

Khol2721 committed:
Delete .gradle directory

Khol2721 committed:
Delete .idea directory

Khol2721 committed:
Delete app directory

Khol2721 committed:
Delete gradle/wrapper directory

Khol2721 committed:
Delete .gitignore

Khol2721 committed:
Delete README.md

Khol2721 committed:
Delete build.gradle

Khol2721 committed:
Delete gradle.properties

Khol2721 committed:
Delete gradlew

Khol2721 committed:
Delete gradlew.bat

Khol2721 committed:
Delete local.properties

Khol2721 committed:
Delete settings.gradle

Khol2721 committed:
fix

Khol2721 committed:
fix

blnvfeip reviewed (commented):
3 - ok
4:
Не помещается на экран кнопка "Выйти"
<img width="548" alt="Screen Shot 2022-01-14 at 11 29 24" src="https://user-images.githubusercontent.com/49048977/149435915-a58bb98b-7143-48bb-9f23-63d09b22307b.png">
5 – не вижу кода пятого дз

Khol2721 committed:
Delete .gradle directory

Khol2721 committed:
Delete .idea directory

Khol2721 committed:
Delete app directory

Khol2721 committed:
Delete gradle/wrapper directory

Khol2721 committed:
Delete README.md

Khol2721 committed:
Delete build.gradle

Khol2721 committed:
Delete gradle.properties

Khol2721 committed:
Delete gradlew

Khol2721 committed:
Delete gradlew.bat

Khol2721 committed:
Delete local.properties

Khol2721 committed:
Delete settings.gradle

Khol2721 committed:
fix

Khol2721 committed:
fix

blnvfeip reviewed (commented):
<img width="1041" alt="Screenshot 2022-01-24 at 19 03 28" src="https://user-images.githubusercontent.com/49048977/150752709-0347ffec-1980-4aab-b554-312beb0a50be.png">
– используйте layout_constraintVertical_bias вместо layout_marginTop

– настройте, пожалуйста, gitignore нормально, не могу смотреть на эту тонну ненужных файлов и изменений с каждым разом

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
fix

не получилось пофиксить .gitignore, при изменениях проект перестает собираться с ошибкой Caused by: org.gradle.api.internal.plugins.PluginApplicationException: Failed to apply plugin 'com.android.internal.application'.

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
test

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
fix

Khol2721 committed:
Delete app/build directory

blnvfeip commented on app/src/main/java/ru/fefu/activitytracker/main_page/UsersActivityDetails.kt:
```diff
@@ -0,0 +1,43 @@
+import ru.fefu.activitytracker.R
+import ru.fefu.activitytracker.databinding.ActivityUsersDetailsFragmentBinding
+
+class UsersActivityDetails(details: ListItem.Item) :
```
нельзя передавать данные в конструктор фрагмента

blnvfeip commented on app/src/main/java/ru/fefu/activitytracker/main_page/MyActivityDetailsFragment.kt:
```diff
@@ -0,0 +1,92 @@
+import ru.fefu.activitytracker.main_page.room.ActivityRoom
+import java.time.*
+
+class MyActivityDetailsFragment(id: Int) :
```
нельзя передавать данные в конструктор фрагмента (2)

blnvfeip reviewed (commented):
такое ощущение что все сговорились и специально копируют код одного и того же человека
